src/pypdfium2/_cli/toc.py

```diff
@@ -24,18 +24,17 @@ def attach(parser):
 def main(args):
     
     pdf = get_input(args)
-    toc = pdf.get_toc(
-        max_depth = args.max_depth,
-    )
+    toc = pdf.get_toc(max_depth=args.max_depth)
     
-    for item in toc:
-        state = "*" if item.n_kids == 0 else "-" if item.is_closed else "+"
-        target = "?" if item.page_index is None else item.page_index+1
+    for bm in toc:
+        count, index, (view_mode, view_pos) = bm.get_count(), bm.get_index(), bm.get_view()
```

Comment:
In the future, `count` and `index` could be inlined below by assignment expression (operator `:=`, requires python >= 3.8).